src/hotspot/share/gc/g1/g1Trace.hpp

```diff
@@ -25,31 +25,21 @@
 #ifndef SHARE_GC_G1_G1TRACE_HPP
 #define SHARE_GC_G1_G1TRACE_HPP
 
-#include "gc/g1/g1GCTypes.hpp"
+#include "gc/g1/g1GCPauseType.hpp"
 #include "gc/shared/gcTrace.hpp"
 
 class G1EvacuationInfo;
 class G1HeapSummary;
 class G1EvacSummary;
 
-class G1YoungGCInfo {
-  G1GCYoungPhase _phase;
-public:
-  G1YoungGCInfo() : _phase(G1GCYoungPhaseEndSentinel) {}
-  void set_phase(G1GCYoungPhase phase) {
-    _phase = phase;
-  }
-  G1GCYoungPhase phase() const { return _phase; }
-};
-
 class G1NewTracer : public YoungGCTracer {
-  G1YoungGCInfo _g1_young_gc_info;
+  G1GCPauseType _pause;
 
 public:
-  G1NewTracer() : YoungGCTracer(G1New) {}
+  G1NewTracer() : YoungGCTracer(G1New), _pause(G1GCPauseType::Invalid) {}
 
   void initialize();
-  void report_yc_phase(G1GCYoungPhase phase);
+  void report_yc_pause(G1GCPauseType pause);
```

Comment:
rename report_yc_pause  to match GCPauseType

Reply:
+1, something like `report_yc_type()` or since this is the `G1NewTracer` just `report_gc_type()`